src/Query/FetchMappedProductsQuery.php

```diff
@@ -26,12 +21,13 @@ public function __construct(private readonly PimCatalogApiClient $catalogApiClie
      */
     public function fetch(string $catalogId): array
     {
+        /** @var array<RawMappedProduct> $rawMappedProducts */
         $rawMappedProducts = $this->catalogApiClient->getMappedProducts($catalogId);
 
         $products = [];
         foreach ($rawMappedProducts as $rawMappedProduct) {
-            $label = isset($rawMappedProduct['title']) && '' !== $rawMappedProduct['title']
-                ? $rawMappedProduct['title']
+            $label = '' !== $rawMappedProduct['name']
+                ? $rawMappedProduct['name']
```

Comment:
Why did you remove the `isset` test ?

Reply:
Because the new JSON schema requires the `name` attribute to be present so we can't have any products without it.